src/pages/SubmissionConfig.js

```diff
@@ -327,7 +338,12 @@ export const SubmissionConfig = ({ methodId }) => {
         displayLaunchModal && h(Modal, {
           title: 'Send submission',
           width: 600,
-          onDismiss: () => { if (!isSubmitting) { setDisplayLaunchModal(false) } },
+          onDismiss: () => {
+            if (!isSubmitting) {
+              setDisplayLaunchModal(false)
+              setWorkflowSubmissionError(undefined)
```

Comment:
Can we set this as `undefined` in its declaration, and save a line of code?

Reply:
It should be default initialized to `undefined`, this is to clear errors after exiting and re-opening the modal